src/OpenSpaceToolkit/Physics/Environment/Magnetic/Earth.cpp

```diff
@@ -186,20 +186,30 @@ Unique<MagneticModel> Earth::Impl::MagneticModelFromType(const Earth::Type& aTyp
     }
     else
     {
-        if (Manager::Get().getMode() == Manager::Mode::Automatic)
+        switch (Manager::Get().getMode())
         {
-            if (!Manager::Get().hasDataFilesForType(aType))
+            case Manager::Mode::Automatic:
             {
-                Manager::Get().fetchDataFilesForType(aType);
+                if (!Manager::Get().hasDataFilesForType(aType))
+                {
+                    Manager::Get().fetchDataFilesForType(aType);
+                }
+                break;
             }
         }
 
-        if (Manager::Get().getMode() == Manager::Mode::Manual)
+        case Manager::Mode::Manual:
         {
             if (!Manager::Get().hasDataFilesForType(aType))
             {
                 throw ostk::core::error::RuntimeError("Cannot load Earth magnetic model, data files are missing.");
             }
+            break;
+        }
+
+        default:
+        {
+            throw ostk::core::error::runtime::Wrong("Manager mode.");
         }
```

Comment:
```suggestion
            }
        }
```
Will still have to re-format this file.